### PR TITLE
Hud Eyepatches

### PR DESCRIPTION
**Pull request by Alekshhh**
## About the PR
Adds hud eyepatches. They're just eyepatches with huds. You can get them from vendors (sectech, medidrobe, bardrobe, one in each)

also tweaks the original eyepatch equip

## Media

image
image
new
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:Alekshhh
- add: Hud eyepatches have been added to BarDrobes, MediDrobes and SecTechs. They're just cooler looking huds.

### CODE REVIEW

**Comment by icekot8** [comment]
looks interesting... but how does it display on arachnids/moths?

**Comment by IProduceWidgets** [comment]
I don't think I like the way it looks on the player.

**Comment by albatrossrus** [comment]
If only could wear two patches at one time... Dreams...

File: Resources/Prototypes/Entities/Clothing/Eyes/hud.yml [hunk 1]
```diff
@@ -216,3 +216,53 @@
...
+  name: beer eyepatch
+  description: A pair of sunHud outfitted with apparatus to scan reagents, as well as providing an innate understanding of liquid viscosity while in motion. For true patriots.
+  components:
+  - type: Sprite
+    sprite: Clothing/Eyes/Hud/beerpatch.rsi
+  - type: Clothing
+    sprite: Clothing/Eyes/Hud/beerpatch.rsi
+  - type: ShowThirstIcons
+  - type: StealTarget
+    stealGroup: ClothingEyesHudBeer
```
Comment (icekot8) [inline Resources/Prototypes/Entities/Clothing/Eyes/hud.yml:267]: I think you can just position the parents differently so it doesn't look so bulky.

For example, replace the parent of ClothingEyesEyepatchHudMedical with ClothingEyesHudMedical. by doing the same with the other eye patches, you'll reduce the amount of text considerably

**Comment by CatMagic1** [comment]
that doesn't... cover the eye

**Comment by CrigCrag** [comment]
> that doesn't... cover the eye
image
spot the eye challenge (spoiler, you can't)

**Comment by Kadeo64** [comment]
would be funny if there was a random chance for a medical one to be present in place of the standard medical hud in med lockers (same with bartender closet)

**Comment by Rainbeon** [comment]
We don't even have MedHUDs in any vendor yet, this feels weird to have in Med Vend without also putting in the base

**Comment by Kadeo64** [comment]
> We don't even have MedHUDs in any vendor yet, this feels weird to have in Med Vend without also putting in the base

they're in the lockers though

**Comment by Alekshhh** [comment]
hey man I used the original eyepatch equipped

File: Resources/Prototypes/Entities/Clothing/Eyes/hud.yml (hunk 1 above)
Comment (Alekshhh) [reply Resources/Prototypes/Entities/Clothing/Eyes/hud.yml:267]: not wrong

**Comment by UbaserB** [comment]
this looks pretty damn based

**Comment by Rainbeon** [comment]
> > We don't even have MedHUDs in any vendor yet, this feels weird to have in Med Vend without also putting in the base
> 
> they're in the lockers though

Some stations have like a single mapped doctor locker

**Comment by Emisse** [comment]
looks too low

**Comment by Alekshhh** [comment]
if I'm changing the equipped then I have to change the original too

**Comment by TheShuEd** [comment]
The black string merges with the eye, shifting it upwards and the 2*2 glass visually lower, and it seems to me that the player turns into some kind of mutant with a crooked face. Don't like it.

**Comment by IProduceWidgets** [comment]
Agreed, I would say change the band color, but that might be weird on the original black one. Might be worth a shot though.

**Comment by TheShuEd** [comment]
Much better now

File: Resources/Textures/Clothing/Eyes/Hud/beerpatch.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/5a73e8f825ff279e82949b9329783a9e3070e2daMade , edited by Alekshhh",
```
Comment (TheShuEd) [inline Resources/Textures/Clothing/Eyes/Hud/beerpatch.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/5a73e8f825ff279e82949b9329783a9e3070e2da Made, edited by Alekshhh",
```

File: Resources/Textures/Clothing/Eyes/Hud/medpatch.rsi/meta.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/5a73e8f825ff279e82949b9329783a9e3070e2daMade , edited by Alekshhh",
```
Comment (TheShuEd) [inline Resources/Textures/Clothing/Eyes/Hud/medpatch.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/5a73e8f825ff279e82949b9329783a9e3070e2da Made, edited by Alekshhh",
```

File: Resources/Textures/Clothing/Eyes/Hud/secpatch.rsi/meta.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/5a73e8f825ff279e82949b9329783a9e3070e2daMade , edited by Alekshhh",
```
Comment (TheShuEd) [inline Resources/Textures/Clothing/Eyes/Hud/secpatch.rsi/meta.json:4]:
```suggestion
  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/5a73e8f825ff279e82949b9329783a9e3070e2da Made, edited by Alekshhh",
```

**Review by TheShuEd — CHANGES_REQUESTED** [review]
some spacing problem

File: Resources/Textures/Clothing/Eyes/Hud/secpatch.rsi/meta.json (hunk 4 above)
Comment (Alekshhh) [reply Resources/Textures/Clothing/Eyes/Hud/secpatch.rsi/meta.json:4]: fug

**Comment by TheShuEd** [comment]
What if we make their number per round no more than 1 per round, and add them to the thief's steal targets? He's got a shortage of things to steal from the medical department.

**Review by TheShuEd — APPROVED** [review]

**Comment by Alekshhh** [comment]
that'd be a different PR

**Pull request merged**